pkg/cmd/task/task_metrics_summary.go

```diff
@@ -13,6 +20,73 @@ func newCmdTaskMetricsSummary(ctx api.Context) *cobra.Command {
 		Short: "Print a table of the key metrics for a given task",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx.Deprecated("Getting the metrics of a task using `dcos task metrics summary` is deprecated.")
+
+			task, err := findTask(ctx, args[0])
+			if err != nil {
+				return err
+			}
+			status := task.Statuses[0]
+			containerID := status.ContainerStatus.GetContainerID().Value
+
+			c := metrics.NewClient(pluginutil.HTTPClient(""))
+			taskMetrics, err := c.Task(task.AgentID.Value, containerID)
+			if err != nil {
+				return err
+			}
+
+			if taskMetrics == nil {
+				return fmt.Errorf("No metrics found for task '%s'", task.TaskID.Value)
+			}
+
+			filteredDatapoints := []metrics.Datapoint{}
+			summaryDatapoints := map[string]float64{
+				"cpus.user_time_secs":      0,
+				"cpus.system_time_secs":    0,
+				"cpus.throttled_time_secs": 0,
+				"mem.limit_bytes":          0,
+				"mem.total_bytes":          0,
+				"disk.used_bytes":          0,
+				"disk.limit_bytes":         0,
+			}
+
+			// Filter the datapoints.
+			for _, datapoint := range taskMetrics.Datapoints {
+				if _, ok := summaryDatapoints[datapoint.Name]; ok {
+					filteredDatapoints = append(filteredDatapoints, datapoint)
+					summaryDatapoints[datapoint.Name] = datapoint.Value
+				}
+			}
+
+			if jsonOutput {
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(filteredDatapoints)
+			}
+
+			table := cli.NewTable(ctx.Out(), []string{"CPU", "MEM", "DISK"})
+			// This value mapping does not make sense (especially for mem and disk) but that is how it was in the Python codebase.
+			cpuUsed := summaryDatapoints["cpus.user_time_secs"] + summaryDatapoints["cpus.system_time_secs"]
+			cpuTotal := cpuUsed + summaryDatapoints["cpus.throttled_time_secs"]
+			if cpuTotal == 0 {
+				cpuTotal = math.MaxFloat64
+			}
+			memUsed := summaryDatapoints["mem.total_bytes"]
+			memTotal := summaryDatapoints["mem.limit_bytes"]
+			if memTotal == 0 {
+				memTotal = math.MaxFloat64
+			}
+			diskUsed := summaryDatapoints["disk.limit_bytes"]
+			diskTotal := summaryDatapoints["disk.used_bytes"]
+			if diskTotal == 0 {
+				diskTotal = math.MaxFloat64
```

Comment:
Shall we print `N/A` here? If the total is missing we can't print a useful percentage.